tests/test_edgeql_functions.py

```diff
@@ -689,3 +689,176 @@ class TestEdgeQLFunctions(tb.QueryTestCase):
         ''', [
             [6.8],
         ])
+
+    @unittest.expectedFailure
+    async def test_edgeql_functions_str_01(self):
+        # at the very least the cast <str> should be equivalent to
+        # a call to str() without explicit format for simple scalars
+        await self.assert_query_result(r'''
+            WITH DT := current_datetime()
+            # FIXME: the cast has a "T" and the str doesn't for some reason
```

Comment:
This is because we actually use a normalization hack in `<str>datetime`:

https://github.com/edgedb/edgedb/blob/6da6f58c933deaabb520053d2672101170eb6ee2/edb/server/pgsql/compiler/typecomp.py#L129-L145